webapp/src/components/Vault/VaultActionsForm/v2/VaultV2DepositWithdrawForm.tsx

```diff
@@ -447,7 +447,11 @@ const VaultV2DepositWithdrawForm: React.FC<VaultV2DepositWithdrawFormProps> = ({
           target="_blank"
           rel="noreferrer noopener"
         >
-          <BridgeButton color={getAssetColor(vaultActionForm.depositAsset)}>
+          <BridgeButton
+            color={getAssetColor(
+              vaultOption === "rsAVAX-THETA" ? "sAVAX" : "WAVAX"
```

Comment:
Could we move this inlined conditional into the `getAssetColor()` function?  Put the hack in one spot